### PR TITLE
In whitch the Slaughter demon gets changed

### PR DESCRIPTION
**Pull request by Aurorablade**
from https://github.com/tgstation/-tg-station/pull/11757, Ports the following:

- A lot of grammar has been changed or improved.
- The events to spawn slaughter demons play a sound to the selected candidates.
- Demon hearts have a different sprite (more red).
-Slaughter demons now store eaten mobs, and will drop them upon death rather than deleting them. Mobs that can eat people but are not slaughter demons will still delete the mob.
-You must have empty hands before using Blood Crawl. In addition, you cannot use anything in your inventory while Blood Crawling.
-Bloodcrawl now has a 2 second delay before emerging/phase in

Things not ported:
- Demon hearts are now activated by eating like normal food.
-Eating a demon heart now replaces your normal heart with it. If the demon heart is removed from your chest, you lose the ability to Blood Crawl, and someone else can eat the heart to gain said ability. (I WOULD LOVE TO DO THIS..but am unsure how...so eh.)

Quirks:
-Animation bug with the added phase in 2 second delay. Need a head smack on how to fix.

Insanity:
-Out of Voodoo Dolls, Guardians/Stands and Swarmers..what should i try to port next.

### CODE REVIEW

**Comment by Dinarzad** [comment]
"-Out of Voodoo Dolls, Guardians/Stands and Swarmers..what should i try to port next."

Swarmers ploz.
They are fun for the whole family, because they are non-lethal antags.

**Comment by tigercat2000** [comment]
`whitch`

**Comment by Aurorablade** [comment]
Changelog:
Edited the title for proper spelling as per @tigercat2000

**Comment by Mijhead** [comment]
Guardian/Stands or go home

File: code/modules/mob/living/bloodcrawl.dm [hunk 1]
```diff
@@ -44,32 +59,44 @@
...
+					src.adjustOxyLoss(-1000)
+					src.adjustToxLoss(-1000)
+
+					if (istype(src, /mob/living/simple_animal/slaughter)) //rason, do not want humans to get this
+
+						var/mob/living/simple_animal/slaughter/demon = src
+						demon.devoured++
+						kidnapped << "<span class='userdanger'>You feel teeth sink into your flesh, and the--</span>"
+						kidnapped.adjustBruteLoss(1000)
+						kidnapped.loc = src
```
Comment (PJB3005) [inline code/modules/mob/living/bloodcrawl.dm]: `forceMove()`

Comment (Aurorablade) [inline code/modules/mob/living/bloodcrawl.dm]: not used to useing forceMove() i know this is storeing the location of the mob/dead person to the demon. I know on DEATH when it drops the eaten people i could use forcemove i think. I need an example as my head is only half awake right now.

Comment (PJB3005) [inline code/modules/mob/living/bloodcrawl.dm]: `forceMove()` should always be used over direct setting of loc, if you don't, things like lighting issues will happen.

**Comment by Earthdivine** [comment]
Swarmers! Adorable and non lethal!

**Comment by Fox-McCloud** [comment]
I vote voodoo dolls---we have a lot of code that's going to be difficult to integrate with swarmers.

**Comment by Aurorablade** [comment]
also this is mergeable unless someone points something else out to me.

**Pull request merged**